package.json

```diff
@@ -147,10 +147,11 @@
     "glob": "^8.0.1",
     "http-server": "^14.1.1",
     "image-webpack-loader": "^8.1.0",
-    "jest": "^26.6.3",
+    "jest": "^29.0.0",
+    "jest-environment-jsdom": "^29.7.0",
```

Comment:
Need to add this as a separate package for jest version `28.0.0` onwards.
https://jest-archive-august-2023.netlify.app/docs/28.x/upgrading-to-jest28#jsdom